src/chains/definitions/zetachainAthensTestnet.ts

```diff
@@ -2,22 +2,29 @@ import { defineChain } from '../../utils/chain/defineChain.js'
 
 export const zetachainAthensTestnet = /*#__PURE__*/ defineChain({
   id: 7001,
-  name: 'ZetaChain Athens Testnet',
+  name: "Zetachain Athens 3 Testnet",
+  testnet:true,
+  iconUrl: "https://s2.coinmarketcap.com/static/img/coins/64x64/21259.png",
   nativeCurrency: {
+    name: "ZetaChain Athens 3 Testnet",
```

Comment:
```suggestion
    name: "ZetaChain Testnet",
```

ZetaChain only has one testnet, so I think it's easier if we just call it "Testnet" without the "Athens" codename.